webportal/package.json

```diff
@@ -74,7 +74,7 @@
     "prebuild": "npm run clean && node config/preprocess.js",
     "build": "webpack --config webpack.config.js --progress --profile",
     "build:dev": "npm run build -- --debug --devtool eval-cheap-module-source-map --output-pathinfo --watch",
-    "prestart": "envsub --env-file .env --system src/app/env.js dist/scripts/env.js",
+    "prestart": "envsub --env-file .env --system src/app/env.js dist/scripts/env.js && ping -c 1 -W 1 172.17.0.1",
```

Comment:
Explain a little?

Reply:
It's a workaround to pass the cluster deploy test in Jenkins, not related to this optimization. I will revert it and raise a new PR to discuss it.